jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.vue

```diff
@@ -112,6 +112,12 @@
           ></v-select>
         </v-row>
 
+        <v-row v-if="current_plot_type==='Radial Profile (Raw)' && subset_area > 5000">

```

Comment:
I have a feeling that this number actually depends on the hardware. Is "5000" representative enough? Should we document how you decided on this number?

Reply:
It likely is, I kind of just guessed, but as I said in the description, I'm hoping that we can revise that during reviews.  Ultimately it isn't a cutoff where we prevent running, just a cutoff where we start displaying the warning, so I think we should err on the side of too low of a number... or maybe we should just always display the message?